Cargo.toml

```diff
@@ -30,6 +25,12 @@ security-framework-sys = "1.0"
 # Dependencies specific to Linux
 [target.'cfg(target_os="linux")'.dependencies]
 procfs = "0.8.0"
+[target.'cfg(any(target_os = "linux", target_os = "unix"))'.dependencies]
+nix = "0.17.0"
+libproc = "0.7.2"
+libc = "0.2.72"
+object = "0.20"
```

Comment:
`object` should remain as dependency on all OSes I think. It is necessary for symbolication in case of remote debugging of unix targets. `gimli` is also necessary for that, so I think that one should also be moved from `cfg(unix)` to all platforms.